docker-compose.yml

```diff
@@ -25,10 +30,16 @@ services:
       EQ_USED_JTI_CLAIM_TABLE_NAME: dev-used-jti-claim
       EQ_DATASTORE_EMULATOR_CREDENTIALS: "True"
       DATASTORE_EMULATOR_HOST: datastore:8432
+      GOOGLE_CLOUD_PROJECT: "local"
       EQ_SUBMISSION_BACKEND: "log"
+      EQ_REDIS_HOST: "redis"
+      EQ_REDIS_PORT: "6379"
+    volumes:
+      - ~/.config/:/root/.config
```

Comment:
This should either be removed or have its scope narrowed.

Reply:
Fixed

Reply:
👍